addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -388,7 +388,7 @@ def _import_fill_invoice_allowance_charge(self, tree, invoice_form, journal, qty
         else:
             return
 
-        xpath = './{*}AllowanceCharge' if is_ubl else './{*}SupplyChainTradeTransaction/{*}ApplicableHeaderTradeSettlement/{*}SpecifiedTradeAllowanceCharge'
+        xpath = './{*}InvoiceLine/{*}AllowanceCharge' if is_ubl else './{*}SupplyChainTradeTransaction/{*}ApplicableHeaderTradeSettlement/{*}SpecifiedTradeAllowanceCharge'
```

Comment:
The AllowanceCharges we are retrieving here are the document's one, the AllowanceCharges on the lines are retrieved in `_import_fill_invoice_line_values`. So I think there is a misunderstanding here.